tests/components/recorder/conftest.py

```diff
@@ -0,0 +1,24 @@
+"""Common test tools."""
+
+import pytest
+
+from homeassistant.components.recorder.const import DATA_INSTANCE
+
+from tests.common import get_test_home_assistant, init_recorder_component
+
+
+@pytest.fixture
+def hass_recorder():
```

Comment:
You can use other fixtures in your fixtures.
```suggestion
async def hass_recorder(hass):
```

Reply:
And then you don't need to deal with the lifecycle of hass etc.